### ISSUE TITLE
Cannot save workflow on scratch due to HDF5 file locking

### ISSUE THREAD

**Issue by maria-yankova** [LightForm-group/UoM-CSF-matflow#3]

**Comment by aplowman** [comment]
For now, we can add `export HDF5_USE_FILE_LOCKING=FALSE` to `.bash_profile`. This should later be addressed in a MatFlow update.

**Comment by SJaffa** [comment]
I think this is fixed now, is that right @aplowman ? If so we can close the issue and remove [this line](https://github.com/LightForm-group/UoM-CSF-matflow/blob/f1ca8c9b61471e0f7b45aec2786c1dae1eff5445/README.md?plain=1#L13) from the setup instructions in the README.

**Comment by aplowman** [comment]
Yes, I think you're right. However, just to note: the UoM-CSF-matflow repo is strictly for the old version of MatFlow. So I would suggest that we add a link at the top of the README pointing towards our new CSF-specific installation instructions, and set that repo to archive mode.
